.github/workflows/ci.yml

```diff
@@ -20,6 +20,53 @@ jobs:
       # Uses sha for added security since tags can be updated
       uses: joschi/setup-jdk@ccd4eb6437f6a33d3c033551a6004cc6f3078307
       with:
-        java-version:  ${{ matrix.java-version }}
+        java-version: ${{ matrix.java-version }}
     - name: Build with Maven
       run: mvn -B package --file pom.xml
+
+  calcite-regression:
+    name: 'regression: Calcite, Java ${{ matrix.java-version }}'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # javac 8 and 11 might produce slightly different bytecode
+        java-version: [8, 11]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 50
+    - name: 'Set up JDK 11'
+      uses: actions/setup-java@v1
```

Comment:
Just in case, if you prefer `joschi/setup-jdk`, then this should probably be replaced.